src/cli/commands/test/iac-local-execution/file-parser.ts

```diff
@@ -32,24 +29,56 @@ const debug = Debug('snyk-test');
 export async function parseFiles(
   filesData: IacFileData[],
   options: IaCTestFlags = {},
+  isTFVarSupportEnabled = false,
 ): Promise<ParsingResults> {
+  let tfFilesData: IacFileData[] = [];
+  let nonTfFilesData: IacFileData[] = [];
```

Comment:
I can refactor it later, but it will be easier if it's done by a bulk rename: 😬 
```suggestion
  let tfFileData: IacFileData[] = [];
  let nonTfFileData: IacFileData[] = [];
```
so that the name follows the type: singular describing a noun